src/common/changeItemAppearence.js

```diff
@@ -0,0 +1,7 @@
+const changeItemAppearence = (e) => {
+  e.currentTarget.childNodes[4].style.display = 'none';
+  e.currentTarget.childNodes[6].style.display = 'block';
+};
+
+/* eslint-disable */
+  export { changeItemAppearence }
```

Comment:
- [ ] As I mentioned in the previous comment(s), Kindly make sure you fix all your `linters` errors instead of disabling them as you did for best practices and clean code.

Reply:
Changed the same